lib/sisimai/bite/email/sendmail.rb

```diff
@@ -0,0 +1,290 @@
+module Sisimai::Bite::Email
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->